embedx/config.schema.json

```diff
@@ -208,6 +208,11 @@
                 }
               ]
             },
+            "can_interrupt": {
```

Comment:
What happens if we set this value and `response.ignore` both to true? Should they be mutually exclusive?

Reply:
Will take a look.

Reply:
Not an JSON schema expert but I think I got this working: 97514c298